### ISSUE TITLE
ELKLayout missing original cause in stacktrace

### ISSUE THREAD

**Issue by Z1kkurat** [TinkoffCreditSystems/tofu#463]
Stacktrace in ELKLayout is missing original error cause. Thus if we catch an error, rethrow it as a cause of new one and log that, there won't be `caused by` for original error.
The reproducer:
```scala
package test

import cats.effect.IO
import tofu.logging.Logs

object Test extends App {
  val logs          = Logs.sync[IO, IO]
  val originalError = new Exception("Valuable info")
  logs
    .byName("Test")
    .flatMap { logging =>
      IO.raiseError(new Exception("Not so valuable info", originalError))
        .handleErrorWith(logging.errorCause("Got an error", _))
    }
    .unsafeRunSync()
}

```
As a result, there won't be a stacktrace of `originalError`, which may be very misleading especially when it can contain valuable information.
Resulting JSON:
```json
{
   "@timestamp":"2020-11-26T13:43:38.375Z",
   "loggerName":"Test",
   "threadName":"main",
   "level":"ERROR",
   "message":"Got an error",
   "exception":"java.lang.Exception: Not so valuable info",
   "stackTrace":"at test.Test$.$anonfun$new$1(Test.scala:12)\nat cats.effect.internals.IORunLoop$.liftedTree3$1(IORunLoop.scala:244)\nat cats.effect.internals.IORunLoop$.step(IORunLoop.scala:244)\nat cats.effect.IO.unsafeRunTimed(IO.scala:327)\nat cats.effect.IO.unsafeRunSync(IO.scala:246)\nat test.Test$.delayedEndpoint$test$Test$1(Test.scala:11)\nat test.Test$delayedInit$body.apply(Test.scala:6)\nat scala.Function0.apply$mcV$sp(Function0.scala:39)\nat scala.Function0.apply$mcV$sp$(Function0.scala:39)\nat scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)\nat scala.App.$anonfun$main$1(App.scala:73)\nat scala.App.$anonfun$main$1$adapted(App.scala:73)\nat scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)\nat scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)\nat scala.collection.AbstractIterable.foreach(Iterable.scala:920)\nat scala.App.main(App.scala:73)\nat scala.App.main$(App.scala:71)\nat test.Test$.main(Test.scala:6)\nat test.Test.main(Test.scala)"
}
```

Formatted stacktrace:
```
at test.Test$.$anonfun$new$1(Test.scala:12)
at cats.effect.internals.IORunLoop$.liftedTree3$1(IORunLoop.scala:244)
at cats.effect.internals.IORunLoop$.step(IORunLoop.scala:244)
at cats.effect.IO.unsafeRunTimed(IO.scala:327)
at cats.effect.IO.unsafeRunSync(IO.scala:246)
at test.Test$.delayedEndpoint$test$Test$1(Test.scala:11)
at test.Test$delayedInit$body.apply(Test.scala:6)
at scala.Function0.apply$mcV$sp(Function0.scala:39)
at scala.Function0.apply$mcV$sp$(Function0.scala:39)
at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)
at scala.App.$anonfun$main$1(App.scala:73)
at scala.App.$anonfun$main$1$adapted(App.scala:73)
at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)
at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)
at scala.collection.AbstractIterable.foreach(Iterable.scala:920)
at scala.App.main(App.scala:73)
at scala.App.main$(App.scala:71)
at test.Test$.main(Test.scala:6)
at test.Test.main(Test.scala)
```

**Comment by ioaoue** [comment]
It was fixed in tofu-0.8.0: https://github.com/TinkoffCreditSystems/tofu/pull/355

**Comment by Odomontois** [comment]
@ioaoue thx
